task-service/openapi_directory.yaml

```diff
@@ -0,0 +1,195 @@
+openapi: 3.0.3
```

Comment:
shouldn't the names of the files be eopnapi_tasks, openapi_users, ... basically in plural according to OAS standards?